xslt/Channels.xsl

```diff
@@ -1,18 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<xsl:stylesheet version="3.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:stylesheet version="1.1" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:ome="http://www.openmicroscopy.org/Schemas/OME/2016-06"
+                ome:schemaLocation="http://www.openmicroscopy.org/Schemas/OME/2016-06/ome.xsd">
 
     <!-- /Metadata/DisplaySetting/Channels/Channel/DyeName => /OME/Image/Channel@Fluorophore -->
-    <xsl:template match="DyeName">
-       <xsl:attribute name="Fluorophore">
-           <xsl:value-of select="."/>
-       </xsl:attribute>
-    </xsl:template>
+<!--    <xsl:template match="DyeName">-->
+<!--       <xsl:attribute name="Fluorophore">-->
+<!--           <xsl:value-of select="."/>-->
+<!--       </xsl:attribute>-->
+<!--    </xsl:template>-->
```

Comment:
Remove if not needed, or fix if desired? Probably need to ask someone on Microscopy is this is desired or not :shrug:

Reply:
Some of this got commented out while I was fighting with validation. Since most of the work is done on it already I would go through one by one and add them seeing if it's possible to get the output to validate with as much of the commented out code added back in. (there's a small chance it'd the 4dn standard not the OME). I wish I could be more help but I honestly can't remember.

Reply:
Resolved in commit 9ee7971c2dea4c4b64ca97158befe3dcf84fb243.